ksp_plugin/interface_monitor.cpp

```diff
@@ -0,0 +1,69 @@
+﻿
+#include "ksp_plugin/interface.hpp
+
+#include <chrono>
+
+#include "quantities/quantities.hpp"
+#include "quantities/si.hpp"
+
+namespace principia {
+namespace interface {
+
+using quantities::Infinity;
+using quantities::Time;
+using quantities::si::Nano;
+using quantities::si::Second;
+
+namespace {
+std::chrono::steady_clock::time_point monitor_start_time;
+bool is_monitor_running = false;
+
+void StartMonitoring() {
```

Comment:
Now that you have specialized interface function I would remove `StartMonitoring` and `StopMonitoring` and move the code to `...MonitorStart` and `...MonitorStop`.